tests/DocumentationTest.php

```diff
@@ -65,6 +65,7 @@ class DocumentationTest extends TestCase
      * annotations that we don’t want documented
      */
     private const INTENTIONALLY_UNDOCUMENTED_ANNOTATIONS = [
+        '@psalm-check-type', // Used internally for testing try-catch-finally, not sure if we want to support it
```

Comment:
should this be left here?

Reply:
Nope, that was before I pulled it out into the separate PR, guess when I merged that in it got left there.